### DISCUSSION TITLE
I cannot even create the first project

### DISCUSSION THREAD

**Discussion by fraberegis** [vendurehq/vendure#918] [Q&A]
Hello,
Sorry if it's a dummy question but neither npx nor npm init works to create my first project to test vendure.
When launching the create command, I have the following error :
Welcome to @vendure/create v1.0.1!

Let's configure a new Vendure project. First a few questions:

✔ Which database are you using? › SQLite
✔ Which programming language will you be using? › JavaScript
✔ Populate with some sample product data? … no / yes
✔ What identifier do you want to use for the superadmin user? … superadmin
✔ What password do you want to use for the superadmin user? … superadmin

Setting up your new Vendure project in /Users/franckraberegis/devs/node/madashop
This may take a few minutes...

  ✖ Installing dependencies
    → An error occurred when installing dependencies. Try running with `--log-level info` or `--log-level verbose` to diagnose.
    Generating app scaffold
    Initializing server
{"message":"An error occurred when installing dependencies. Try running with `--log-level info` or `--log-level verbose` to diagnose.","command":"yarnpkg add --exact --ignore-engines @vendure/core @vendure/email-plugin @vendure/asset-server-plugin @vendure/admin-ui-plugin better-sqlite3 --cwd /Users/franckraberegis/devs/node/madashop","context":{}}
npm ERR! code 1
npm ERR! path /Users/franckraberegis/devs/node
npm ERR! command failed
npm ERR! command sh -c create madashop verbose

npm ERR! A complete log of this run can be found in:
[2021-06-04T11_34_00_246Z-debug.log](https://github.com/vendure-ecommerce/vendure/files/6599209/2021-06-04T11_34_00_246Z-debug.log)

**Comment by CliffCrerar** [comment]
Good Day

While in the console. do not write anything to do with init. Make sure the directory for your project does not event exist.

No init of any kind. 

## > A vendure project is bootstrapped differently to the usual npm project

(while anywhere you like in your file system)

- run:

`npx @vendure/create <your-project-name>`

- Your project should start creating itself now and once done, run the following 

`cd  <your-project-name>`

- You are now in the root of your backend project.

- Re your database question, vendure ships with connectors for `MySQL` , `PostgreSQL`, `MariaDb` and `SQLite`.

> NOTE: Do not attempt to connect to a data source where SSL is required. This wont work on startup but can be configured afterward.

- I used PostreSQL container to connect a database to create the demo database.

`docker run --rm -it  -e POSTGRES_PASSWORD="<strong passwd>" -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432/tcp postgres:latest`

**Reply by fraberegis** [reply]
I am facing the exact same error

| => npx @vendure/create madashop

Welcome to @vendure/create v1.0.1!

Let's configure a new Vendure project. First a few questions:

✔ Which database are you using? › SQLite
✔ Which programming language will you be using? › JavaScript
✔ Populate with some sample product data? … no / yes
✔ What identifier do you want to use for the superadmin user? … superadmin
✔ What password do you want to use for the superadmin user? … superadmin

Setting up your new Vendure project in /Users/franckraberegis/devs/node/madashop
This may take a few minutes...

  ✖ Installing dependencies
    → An error occurred when installing dependencies. Try running with `--log-le
…
    Generating app scaffold
    Initializing server
{"message":"An error occurred when installing dependencies. Try running with `--log-level info` or `--log-level verbose` to diagnose.","command":"yarnpkg add --exact --ignore-engines @vendure/core @vendure/email-plugin @vendure/asset-server-plugin @vendure/admin-ui-plugin better-sqlite3 --cwd /Users/franckraberegis/devs/node/madashop","context":{}}
npm ERR! code 1
npm ERR! path /Users/franckraberegis/devs/node
npm ERR! command failed
npm ERR! command sh -c create madashop

npm ERR! A complete log of this run can be found in:

**Reply by CliffCrerar** [reply]
Which version of `node.js` are your running? 

Are you using windows, Mac or linux?

**Reply by fraberegis** [reply]
Hi,

I am on MacOs BigSur 11.3, Macbook Pro (2016)
node v15.5.0
npm 7.3.0

**Reply by michaelbromley** [reply]
@fraberegis Can you try this with either Node v14.x or v16.x and see if the issue persists?

**Comment by fraberegis** [comment]
OK.
Installed latest (16.x)
Facing this:
| => npx @vendure/create madashop

Welcome to @vendure/create v1.0.1!

Let's configure a new Vendure project. First a few questions:

✔ Which database are you using? › SQLite
✔ Which programming language will you be using? › JavaScript
✔ Populate with some sample product data? … no / yes
✔ What identifier do you want to use for the superadmin user? … superadmin
✔ What password do you want to use for the superadmin user? … superadmin

Setting up your new Vendure project in /Users/franckraberegis/devs/node/madashop
This may take a few minutes...

  ⠧ Installing dependencies
    → Installing concurrently
    Generating app scaffold
    Initializing server
TypeError: tslib_1.__spreadArray is not a function
    at _matchUntil (/Users/franckraberegis/devs/node/madashop/node_modules/juice/node_modules/cheerio/lib/api/traversing.js:130:30)
    at Object.<anonymous> (/Users/franckraberegis/devs/node/madashop/node_modules/juice/node_modules/cheerio/lib/api/traversing.js:212:24)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
  ✔ Installing dependencies
  ✔ Generating app scaffold
  ✖ Initializing server
    → tslib_1.__spreadArray is not a function
{"context":{"configFile":"/Users/franckraberegis/devs/node/madashop/src/vendure-config.js"}}

**Reply by michaelbromley** [reply]
@fraberegis you're unlucky in that this issue just popped up in the past 2 days due to a deep dependency change, see https://github.com/vendure-ecommerce/vendure/issues/925#issuecomment-858383970

Getting it fixed asap.
